pkg/operator/ceph/controller/spec.go

```diff
@@ -388,15 +388,18 @@ func AppLabels(appName, namespace string) map[string]string {
 }
 
 // CephDaemonAppLabels returns pod labels common to all Rook-Ceph pods which may be useful for admins.
-// App name is the name of the application: e.g., 'rook-ceph-mon', 'rook-ceph-mgr', etc.
+// App name is the name of the application: e.g., 'rook-ceph-mon', 'rook-ceph-mgr', etc
 // Daemon type is the Ceph daemon type: "mon", "mgr", "osd", "mds", "rgw"
 // Daemon ID is the ID portion of the Ceph daemon name: "a" for "mon.a"; "c" for "mds.c"
-func CephDaemonAppLabels(appName, namespace, daemonType, daemonID string, includeNewLabels bool) map[string]string {
+// ResourceName is the resource metadata.name: rook-ceph-mon-a", "rook-ceph-osd-1", etc
+// ResourceKind is the CR type: "CephCluster", "CephFilesystem", etc
+func CephDaemonAppLabels(appName, namespace, daemonType, daemonID, resourceName, resourceKind string, includeNewLabels bool) map[string]string {
```

Comment:
Need the parentName to pass to AddRecommendedLabels()
```suggestion
func CephDaemonAppLabels(appName, namespace, parentName, daemonType, daemonID, resourceName, resourceKind string, includeNewLabels bool) map[string]string {
```